README.md

```diff
@@ -2,15 +2,20 @@
 
 NOT ready for use.
 
-When it is, usage will look like:
+Current test procedure:
```

Comment:
We should note the requirement that the project be tracked with git. The steps below might make someone things it's simply suggested.